src/couch/src/couch_bt_engine.erl

```diff
@@ -141,10 +141,10 @@ delete_compaction_files(RootDir, FilePath, DelOpts) ->
         [".compact", ".compact.data", ".compact.meta"]
     ).
 
-is_compacting(DbName) ->
+is_compacting(FilePath) ->
```

Comment:
As an extra precaution to emphasize this is a file path and not a binary dbname, let's assert that `FilePath` is a non-empty list `[_ | _] = FilePath`